dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManagerTest.java

```diff
@@ -119,8 +111,8 @@
     private JdbcEventAnalyticsTableManager subject;
 
     private BeanRandomizer rnd = new BeanRandomizer();
-
-    private static final Date START_TIME = new DateTime( 2019, 8, 1, 0, 0 ).toDate();
+    private int currentYear = getCurrentYear();
```

Comment:
Move the `getCurrentYear` call into `setUp` method to initialization time explicit.